### PR TITLE
feat(integrations): add support for google play

### PR DESCRIPTION
**Pull request by hassan254-prog**
## Describe your changes

- Add support for Google Play

## Issue ticket number and link
[EXT-126](https://linear.app/nango/issue/EXT-126/add-support-for-google-play)

### CODE REVIEW

**Review by TBonnin — COMMENTED** [review]
is the logo corrrect? https://duckduckgo.com/?t=ffab&q=google+play+logo&atb=v404-1&ia=web

**Comment by hassan254-prog** [comment]
@TBonnin , for the Google Play API, I found it a bit confusing to choose between the Google Play Console logo and the Play Store logo. The Play Console logo is more aligned with developer tools and app management, while the Play Store logo is more consumer-focused. Please let me know if we should change it.

File: packages/webapp/public/images/template-logos/google-play.svg
Comment (bodinsamuel) [inline packages/webapp/public/images/template-logos/google-play.svg:1]: They are not even consistent in the doc. I would favor the most expected one 👌🏻

**Review by bodinsamuel — APPROVED** [review]

**Review by TBonnin — APPROVED** [review]
wrt the logo not a big deal we can always change it later if needed

**Pull request merged**